src/components/blockHelpers.tsx

```diff
@@ -250,16 +250,31 @@ export const getMinimalColorStyles: ColorSetGetter<minimalColorComponentColors>
 export const getSectionSizeStyle = (size: SectionSize) =>
   sectionSizeStyles[size]
 
-export const backgroundImageStyles = (backgroundImage: string) => {
-  return (
-    backgroundImage !== 'none' &&
-    backgroundImage !== '' && {
-      backgroundImage: `url(${backgroundImage})`,
-      backgroundSize: 'cover',
-      backgroundPosition: 'center center',
-      backgroundRepeat: 'no-repeat',
-    }
-  )
+export const backgroundImageStyles = (
+  backgroundImage = '',
+  backgroundImageMobile = '',
+  tint = false,
+) => {
+  if (backgroundImage === '') {
+    return
+  }
+
+  const bgTint = tint
+    ? 'linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), '
+    : ''
+
+  return {
+    backgroundImage: `${bgTint}url(${backgroundImage})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center center',
+    backgroundRepeat: 'no-repeat',
+
+    [TABLET_BP_DOWN]: {
+      backgroundImage: `${bgTint}url(${
+        backgroundImageMobile === '' ? backgroundImage : backgroundImageMobile
+      })`,
```

Comment:
Might not be the most beautiful solution, would like to omit this css completely if there is no `backgroundImageMobile`

Reply:
what about something like

```ts
[TABLET_BP_DOWN]: backgroundImageMobile ? {
  backgroundImage: ...
} : {},
```

Reply:
Ah yes, thank you! 🙏

Reply:
Updated: https://github.com/HedvigInsurance/web-next/pull/458/commits/0de1df27ff13f31bf691d9653c433c163fd897eb